certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2420 @@
+      "url": "https://support.talkdesk.com/",
+      "email": "support@talkdesk.com"
+    }
+  },
+  "host": "api.talkdeskstg.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/webhooks/schema/{listID}": {
+      "get": {
+        "description": "Schema for selected item in action.",
+        "summary": "Item schema",
+        "operationId": "GetListSchema",
+        "parameters": [
+          {
+            "name": "listID",
+            "type": "string",
+            "in": "path",
+            "description": "Item id.",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Item id."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/contacts/{contact_id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Data response for this action.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "contact": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "Contact ID.",
+                      "title": "Id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "Contact name.",
+                      "title": "Name"
+                    },
+                    "company": {
+                      "type": "string",
+                      "description": "Contact company.",
+                      "title": "Company"
+                    },
+                    "phones": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "label": {
+                            "type": "string",
+                            "description": "Phone number label.",
+                            "title": "Label"
+                          },
+                          "number": {
+                            "type": "string",
+                            "description": "Phone number in E.164 format.",
+                            "title": "Number"
+                          }
+                        }
+                      },
+                      "description": "The list of the contact's phones.",
+                      "title": "Phones"
+                    }
+                  },
+                  "description": "The contact data.",
+                  "title": "Contact"
+                }
+              }
+            }
+          },
+		  "default": {
```

Comment:
@StefanCRT22 Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**